docs/faststore/docs/customization/building-sections/list-of-native-sections.mdx

```diff
@@ -9,7 +9,33 @@ Sections in FastStore are components that organize and encapsulate other compone
 
 For example, the [Hero](https://developers.vtex.com/docs/guides/faststore/organisms-hero) is a native section designed to capture your brand's identity and messaging. To effective function in the store, the Hero comprises three components: `Hero`, `HeroImage`, and `HeroHeader`.
 
----
+## Considerations while customizing default components
+
+Customizing `__experimentalProductCard` and `SearchInput` can impact store analytics. These components collect data for your store analytics. Prioritize maintaining tracking functionality.
+
+### `__experimentalProductCard`
+
+**Affected sections:** [`ProductGallery`](https://developers.vtex.com/docs/guides/faststore/building-sections-list-of-native-sections#productgallery) and [`ProductShelf`](https://developers.vtex.com/docs/guides/faststore/building-sections-list-of-native-sections#productshelf).
```

Comment:
```suggestion
**Impacted sections:** [`ProductGallery`](https://developers.vtex.com/docs/guides/faststore/building-sections-list-of-native-sections#productgallery) and [`ProductShelf`](https://developers.vtex.com/docs/guides/faststore/building-sections-list-of-native-sections#productshelf).
```

How about using Impacted instead of Affected?